src/main/java/org/manifold/compiler/back/digital/smt2/QFABV.java

```diff
@@ -0,0 +1,81 @@
+package org.manifold.compiler.back.digital.smt2;
+
+
+// Code generation helpers for SMT2 QF_ABV expressions
+public class QFABV {
```

Comment:
I don't know enough about the output notation to effectively review this, but I trust that they're probably right XD